app/views/application/_admin_menu.html.erb

```diff
@@ -0,0 +1,22 @@
+<button type="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+<nav>
+  <ul id="navigation" class="govuk-header__navigation " aria-label="Top Level Navigation">
+    <% if service_operator_signed_in? %>
+      <li class="govuk-header__navigation-item">
+        <%= link_to "View claims", admin_claims_path, class: "govuk-header__link" %>
+      </li>
+      <li class="govuk-header__navigation-item">
+        <%= link_to "Search", search_admin_claims_path, class: "govuk-header__link" %>
+      </li>
+      <li class="govuk-header__navigation-item">
+        <%= link_to "Payroll", admin_payroll_runs_path, class: "govuk-header__link" %>
+      </li>
+      <li class="govuk-header__navigation-item">
+        <%= link_to "Open/close a service", admin_policy_configurations_path, class: "govuk-header__link" %>
+      </li>
+    <% end %>
+    <li class="govuk-header__navigation-item">
```

Comment:
Should this be in the if statement? Can they sign out if they are not signed in? 🤔

Reply:
No, we always show the "sign out" link for a signed in user, but we only show the other links if the user is signed in as a "service operator". e.g. Cantium users don't get to see those links but should be able to sign-out still.